src/plugins/GrabThrowController.ts

```diff
@@ -155,15 +154,14 @@ export function GrabThrowSystem(ctx: GameState) {
     const s: Vector3 = new Vector3().fromArray(source);
     const t: Vector3 = new Vector3().fromArray(target);
```

Comment:
Out of scope of this PR but you may as well fix these extra allocations while you're at it

Reply:
this is only on-click but may as well